angular-workspace/projects/ni/nimble-angular/label-provider/table/nimble-label-provider-table-with-defaults.directive.ts

```diff
@@ -17,5 +17,11 @@ export class NimbleLabelProviderTableWithDefaultsDirective {
         this.elementRef.nativeElement.groupsCollapseAll = $localize`:Nimble table - collapse all groups|:Collapse all groups`;
         this.elementRef.nativeElement.cellActionMenu = $localize`:Nimble table - cell action menu|:Options`;
         this.elementRef.nativeElement.columnHeaderGrouped = $localize`:Nimble table - column header grouped indicator|:Grouped`;
+        this.elementRef.nativeElement.columnHeaderSortedAscending = $localize`:Nimble table - column header sorted ascending indicator|:Sorted ascending`;
+        this.elementRef.nativeElement.columnHeaderSortedDescending = $localize`:Nimble table - column header sorted descending indicator|:Sorted descending`;
```

Comment:
"indicator" should be removed from these and elsewhere in the PR and for header grouped generally